NEWS

```diff
@@ -39,25 +39,21 @@ CHANGES WITH 252 in spe:
 
         New Features:
 
-        * systemd-measure is a new tool for precalculating and signing expected
-          TPM2 PCR values seen once a given unified kernel image (UKI) with
-          systemd-stub is booted. This is useful for implementing TPM2 policies
-          for LUKS encrypted volumes and encrypted system/service credentials,
-          that robustly bind to kernels carrying appropriate PCR signature
-          information. The signed expected PCR information, and the public key
-          used for the signature may be embedded inside UKIs for this purpose,
-          so that it is automatically available in userspace, once the UKI is
-          booted.
-
-          systemd-cryptsetup, systemd-cryptenroll and systemd-creds have been
+        * systemd-measure is a new tool for calculating and signing expected
+          TPM2 PCR values for a given unified kernel image (UKI) booted via
+          sd-stub. The public key used for the signature and the signed
+          expected PCR information can be embedded inside the UKI. This
+          information can be read from the UKI binary and is also made
+          available to userspace in the booted kernel.
```

Comment:
"read from the UKI binary"? what is that supposed to mean? from the stub?

Reply:
The stub does this. But it's also possible to ready from any other tool. In the future tools can use this to construct TPM policies, but even right not one could check that the values are as expected after booting the kernel.

Reply:
To add a bit to this: this just describes what has been added. It doesn't necessarily say what the best way to use it. That is described later. I think it is important to describe what is being added in a general way first, so that things are not "magic", but the reader can figure out how a new feature works.

Reply:
ah, so "read from the binary" is about something else extracting it from the binary. I understood this to mean that something in the binary is the subject that reads it.

Maybe say "extracted from the UKI by external tools and code in the image itself"?

Reply:
Updated. I added another commit for some man page strings too.